lib/cisco_node_utils/feature.rb

```diff
@@ -151,6 +151,26 @@ def self.vni_enabled?
     end
 
     # ---------------------------
+    def self.vtp_enable
+      return if vtp_enabled?
+      result = config_set('feature', 'vtp', state: '')
+      cli_error_check(result)
+    end
+
+    def self.vtp_disable
+      return unless vtp_enabled?
+      config_set('feature', 'vtp', state: 'no')
```

Comment:
Please add a comment to this one explaining why it exists, i.e. why it's a special case. I don't want others to start adding disable methods to this file.